vocab-importer/src/services/processors/json_processor.py

```
@@ -0,0 +1,11 @@
+import json
+from .base import FileProcessor
+from ...schemas.vocabulary import VocabularyImport
+
+class JsonProcessor(FileProcessor):
+    async def process(self, file: UploadFile) -> Dict[str, Any]:
```

Ruff (0.8.2)

Undefined name `UploadFile`

(F821)